src/Capgemini.PowerApps.PackageDeployerTemplate/PackageTemplateBase.cs

```diff
@@ -448,12 +462,24 @@ private void ExecuteLifecycleEvent(string eventName, Action eventAction)
 
         private void LogLifecycleEventStart(string lifecycleEvent)
         {
-            this.PackageLog.Log($"{nameof(PackageTemplateBase)}.{lifecycleEvent} running...", TraceEventType.Information);
+            this.TraceLoggerAdapter.LogInformation($"{nameof(PackageTemplateBase)}.{lifecycleEvent} running...");
         }
 
         private void LogLifecycleEventEnd(string lifecycleEvent)
         {
-            this.PackageLog.Log($"{nameof(PackageTemplateBase)}.{lifecycleEvent} completed.", TraceEventType.Information);
+            this.TraceLoggerAdapter.LogInformation($"{nameof(PackageTemplateBase)}.{lifecycleEvent} completed.");
+        }
+
+        private void LogTaskCompleteResult()
```

Comment:
Will this double log? e.g.

`##[task.logissue type=error]##vso[task.complete result=Failed;]DONE`

Reply:
yup, good spot!